blocks/columns/columns.css

```diff
@@ -3,6 +3,20 @@
     flex-direction: column;
     }
 
+.columns-container h2 {
```

Comment:
Can this be `.columns h4` or are you targeting `h4` outside of the Columns block as well?

Reply:
No, this is for all H2's that are in columns. All the columns that have an H2 needed margin.